src/diffusers/models/unet_3d_condition.py

```diff
@@ -104,6 +115,16 @@ def __init__(
         norm_eps: float = 1e-5,
         cross_attention_dim: int = 1024,
         attention_head_dim: Union[int, Tuple[int]] = 64,
+        dual_cross_attention: bool = False,
```

Comment:
All the following should not be added - let's try to keep the `__init__` as clean as possible :-)

Reply:
Sure Patrick